packages/ariakit-playground/src/playground-editor.tsx

```diff
@@ -133,11 +138,16 @@ export const usePlaygroundEditor = createHook<PlaygroundEditorOptions>(
     const [editable, setEditable] = useState(false);
     const [focusVisible, setFocusVisible] = useState(false);
 
+    const [expanded, setExpanded] = useControlledState(
+      defaultExpanded ?? !maxHeight,
+      expandedProp,
+      setExpandedProp
+    );
+
```

Comment:
Defining the state here so we can access the expanded state in the `PlaygroundEditor` component. `PlaygroundEditor` uses `PlaygroundCode` underneath and extends its prop types. Since `PlaygroundEditor` can also receive the expanded props, we're doing the same thing here as in the `PlaygroundEditor` component.